### PR TITLE
[PP-37] Add posibility to sign out from app

### PR DESCRIPTION
**Pull request by MakFill**
Task: https://andrei-lysiuk.atlassian.net/browse/PP-37
Refactored Issues layout and add profile btn with modal with posibility to sign out from app

### CODE REVIEW

File: src/containers/ProfileInfo/ProfileInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <div className={styles.nameRoleWrapper}>
+              <h4>
+                {name}
+              </h4>
+              <p className={styles.role}>
+                {role || 'Member'}
+              </p>
+            </div>
+          </div>
+          <div
```
Comment (mrakofff) [inline src/containers/ProfileInfo/ProfileInfo.tsx]: Why it's not a button?

File: src/containers/ProfileInfo/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export {} from './ProfileInfo';
```
Comment (mrakofff) [inline src/containers/ProfileInfo/index.ts]: ?

File: src/components/Issues/Issues.tsx [hunk 3]
```diff
@@ -51,14 +64,37 @@ export const Issues: React.FC = () => {
...
+      <button
+        className={styles.issuesListWrapper}
+        onClick={issuesListIconHandler}
+      >
+        <div className={styles.issuesList} />
+      </button>
+      {isIssuesShow
+      && (
+        <div
+          className={styles.issuesWrapper}
```
Comment (mrakofff) [inline src/components/Issues/Issues.tsx:77]: Why not `button` tag?

**Review by mrakofff — CHANGES_REQUESTED** [review]

File: src/containers/ProfileInfo/index.ts (hunk 2 above)
Comment (MakFill) [reply src/containers/ProfileInfo/index.ts]: fixed

File: src/components/Issues/Issues.tsx (hunk 3 above)
Comment (MakFill) [reply src/components/Issues/Issues.tsx:77]: fixed

File: src/containers/ProfileInfo/ProfileInfo.tsx (hunk 1 above)
Comment (MakFill) [reply src/containers/ProfileInfo/ProfileInfo.tsx]: fixed

**Review by andreikaptsiuh — APPROVED** [review]

File: src/store/actions/membersActions.ts [hunk 4]
```diff
@@ -5,4 +5,5 @@ export const membersActions = {
   addRoomMembers: createAction<IUser[]>('[MEMBERS]:addRoomMembers'),
   addRoomMember: createAction<IUser>('[MEMBERS]:addRoomMember'),
   deleteRoomMember: createAction<IUser['userId']>('[MEMBERS]:deleteRoomMember'),
+  signOut: createAction('[MEMBERS]:signOut'),
```
Comment (mrakofff) [inline src/store/actions/membersActions.ts]: remove redundant actions from reducers

**Review by mrakofff — APPROVED** [review]

File: src/store/actions/membersActions.ts (hunk 4 above)
Comment (MakFill) [reply src/store/actions/membersActions.ts]: removed

**Pull request merged**